data_safe_haven/infrastructure/components/wrapped/nfsv3_storage_account.py

```diff
@@ -0,0 +1,68 @@
+from collections.abc import Mapping, Sequence
+
+from pulumi import Input, Output, ResourceOptions
+from pulumi_azure_native import storage
+
+from data_safe_haven.external import AzureIPv4Range
+
+
+class NFSV3StorageAccount(storage.StorageAccount):
+    encryption_args = storage.EncryptionArgs(
```

Comment:
Is there any reason for these to be a class-level attribute rather than living in the `super().__init__` call?

Reply:
I don't think it's a problem. Since these are all static properties anyway, it shouldn't matter if they're shared between instances or not.